Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -3240,6 +3241,39 @@ CSSRule* Parser::convert_to_rule(NonnullRefPtr<Rule> rule)
 
             return CSSKeyframesRule::create(m_context.realm(), name, move(keyframes)).release_value_but_fixme_should_propagate_errors();
         }
+        if (rule->at_rule_name().equals_ignoring_ascii_case("namespace"sv)) {
+            // https://drafts.csswg.org/css-namespaces/#syntax
+            auto token_stream = TokenStream { rule->prelude() };
+            dbgln("Parsing namespace");
+            auto transaction = token_stream.begin_transaction();
+            token_stream.skip_whitespace();
+
+            auto token = token_stream.peek_token();
+            Optional<StringView> prefix = {};
+            if (token.is(Token::Type::Ident)) {
+                prefix = token.token().ident();
+                token_stream.next_token();
+                token_stream.skip_whitespace();
+                token = token_stream.peek_token();
+            }
+
+            DeprecatedString namespace_uri;
+            if (token.is(Token::Type::String)) {
+                namespace_uri = token.token().string();
+            } else if (token.is_function()) {
+                Optional<AK::URL> url = parse_url_function(token, AllowedDataUrlType::None);
+                if (url.has_value()) {
```

Comment:
Confusingly, `url()` isn't always a function. It might also be the special `<url>` token. So just check if `parse_url_function()` succeeds instead.